music/models.py

```diff
@@ -36,6 +36,7 @@ def __unicode__(self):
 class TemporaryMusic(models.Model):
     music_id = models.CharField(max_length=16)
     name = models.CharField(max_length=255)
+    channel_name = models.CharField(max_length=255, null=True)
```

Comment:
No,field=models.CharField(max_length=255) and in the migration model, you can force a default value for old entries that were in database before.
manage.py makemigration asks you this if you didn't set null=True